.github/workflows/examples.yml

```diff
@@ -35,4 +35,25 @@ jobs:
       - name: install tflint
         run: go install github.com/terraform-linters/tflint@v0.31.0
       - name: terraform
-        run: make tflint
\ No newline at end of file
+        run: |
+          for DIR in $(find ./examples -type f -name '*.tf' -exec dirname {} \; | sort -u); do
+              pushd "${DIR}"
+              if [ -f terraform.template.tfvars ]; then
+                cp terraform.template.tfvars terraform.tfvars
+              fi
+              echo; echo -e "\e[1;35m===> Initializing Example: $DIR <===\e[0m"; echo
+              terraform init
+              echo; echo -e "\e[1;35m===> Format Checking Example: $DIR <===\e[0m"; echo
+              terraform fmt -check
+              echo; echo -e "\e[1;35m===> Validating Example: $DIR <===\e[0m"; echo
+              # Catch errors
+              terraform validate
+              # Terraform syntax checks
+              tflint \
+                --enable-rule=terraform_deprecated_interpolation \
+                --enable-rule=terraform_deprecated_index \
+                --enable-rule=terraform_unused_declarations \
+                --enable-rule=terraform_comment_syntax \
+                --enable-rule=terraform_required_version
```

Comment:
@thetonymaster any input here?

Reply:
I'll add `terraform_typed_variables` and `terraform_unused_declarations` too. Just to have cleaner code

Reply:
@thetonymaster are you asking for a change in this PR, or does the "I'll add" mean you will submit a PR to add these after this is merged?

Reply:
Wrong verb, "`I'd` add"

Reply:
I already added `terraform_unused_declarations`, I'll try to add `terraform_typed_variables` on a separate PR that may be a big change as we are not typing currently and I'd rather merge this sooner rather than later